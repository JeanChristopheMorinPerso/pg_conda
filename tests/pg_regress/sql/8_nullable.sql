BEGIN;
CREATE EXTENSION pg_conda;

SELECT NULL::condaversion;

ROLLBACK;
