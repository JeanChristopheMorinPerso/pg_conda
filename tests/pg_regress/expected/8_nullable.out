BEGIN;
CREATE EXTENSION pg_conda;
SELECT NULL::condaversion;
 condaversion 
--------------
 
(1 row)

ROLLBACK;
